version.py

```diff
@@ -1,4 +1,14 @@
+import os
+
 ZULIP_VERSION = "2.0.3+git"
+# Add information on number of commits and commit hash to version, if available
+zulip_git_version_file = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'var', 'zulip-git-version')
+if os.path.exists(zulip_git_version_file):
+    with open(zulip_git_version_file) as f:
+        version = f.read().strip()
+        if version:
+            ZULIP_VERSION = version
```

Comment:
I guess with this implementation, in the development environment it'll always read `2.0.3+git`, but in production it'll have the more expressive value?  I think that's probably the best option, but perhaps we should add a comment explaining that detail?

Reply:
Yeah, that is correct. I'll add a comment. Also, when running from `git` the version reads something like `2.0.0-1213-g6d4c84339` and not `2.0.3-xxxx`. 
I'm not sure if this acceptable, or we want to get the version number differently to fix this.

Reply:
I'm fine with those having a different look; what matters is that it's clear what the version means, and I think that'll be the case in all situations.